mep/books/templates/books/snippets/work_result.html

```diff
@@ -4,7 +4,7 @@
         {# title - solr only #}
         <div class="title">
             <dt>title</dt>
-            <dd>{{ work.title }}</dt>
+            <dd>{{ work.title.0 }}</dt>
```

Comment:
This is because it's a multivalued field now? Will that cause problems if we want to share this template with the book detail page, or is there not enough overlap to share them?

Reply:
yep. I don't think so; we do something with members where we basically check what page we're on and choose the value to show based on that. It's not the most elegant, but it works fine. If there isn't enough overlap based on recent changes to the book detail page, though, I won't force it to share the snippet.